sdk/digitaltwins/ci.yml

```diff
@@ -30,3 +30,5 @@ extends:
     Artifacts:
     - name: azure_mgmt_digitaltwins
       safeName: azuremgmtdigitaltwins
+    - name: azure_digitaltwins_core
+      safeName: azuredigitaltwinscore
```

Comment:
Once it's in a separate folder, we will also need to add `azure-digitaltwins-nspkg` to these Artifacts

Reply:
Fixed